internal/kafka/internal/workers/cluster_mgrs/dynamic_scaleup_mgr.go

```diff
@@ -51,9 +57,64 @@ func (m *DynamicScaleUpManager) Reconcile() []error {
 	glog.Infoln("running dynamic scale up reconcile event")
 	defer m.logReconcileEventEnd()
 
-	return nil
+	var errs []error
+	errs = m.reconcileClustersForRegions()
+
+	return errs
 }
 
 func (m *DynamicScaleUpManager) logReconcileEventEnd() {
 	glog.Infoln("dynamic scale up reconcile event finished")
 }
+
+// reconcileClustersForRegions creates an OSD cluster for each supported cloud provider and region where no cluster exists.
+func (m *DynamicScaleUpManager) reconcileClustersForRegions() []error {
+	var errs []error
```

Comment:
This method originally had a check to verify whether dynamic scaling was enabled or not.
The check has been removed now because `reconcileClustersForRegions` is only called when dynamic scaling is enabled, from the Reconcile method of the  dynamic scale up manager

Reply:
makes sense